linters/detekt/plugin.yaml

```diff
@@ -14,8 +14,9 @@ lint:
       commands:
         - name: lint
           output: sarif
+          # TODO(Tyler): This needs to be platform-dependent
           run:
-            detekt-cli --build-upon-default-config --config .detekt.yaml --input ${target,} --report
+            detekt-cli.bat --build-upon-default-config --config .detekt.yaml --input ${target,} --report
```

Comment:
We already search for .exe in the CLI, we need to extend it to also look for bat and then we can revert this.